diff --git a/node_modules/starknet/dist/index.d.ts b/node_modules/starknet/dist/index.d.ts
index f61aa2b..1dcf6c6 100644
--- a/node_modules/starknet/dist/index.d.ts
+++ b/node_modules/starknet/dist/index.d.ts
@@ -1826,6 +1826,7 @@ type ReadMethods = {
     starknet_estimateFee: {
         params: {
             request: BROADCASTED_TXN[];
+            simulation_flags: Array<SIMULATION_FLAG$1>;
             block_id: BLOCK_ID;
         };
         result: FeeEstimate[];
diff --git a/node_modules/starknet/dist/index.js b/node_modules/starknet/dist/index.js
index 1cc5884..2bbaafb 100644
--- a/node_modules/starknet/dist/index.js
+++ b/node_modules/starknet/dist/index.js
@@ -3031,6 +3031,7 @@ var RpcProvider = class {
     );
     return this.fetchEndpoint("starknet_estimateFee", {
       request: [transaction],
+      simulation_flags: [],
       block_id
     }).then(this.responseParser.parseFeeEstimateResponse);
   }
diff --git a/node_modules/starknet/dist/index.mjs b/node_modules/starknet/dist/index.mjs
index 0d96902..d933ed8 100644
--- a/node_modules/starknet/dist/index.mjs
+++ b/node_modules/starknet/dist/index.mjs
@@ -1,4 +1,4 @@
-var __defProp = Object.defineProperty;
+svar __defProp = Object.defineProperty;
 var __export = (target, all) => {
   for (var name in all)
     __defProp(target, name, { get: all[name], enumerable: true });
@@ -2933,6 +2933,7 @@ var RpcProvider = class {
     );
     return this.fetchEndpoint("starknet_estimateFee", {
       request: [transaction],
+      simulation_flags: [],
       block_id
     }).then(this.responseParser.parseFeeEstimateResponse);
   }
@@ -2948,6 +2949,7 @@ var RpcProvider = class {
     );
     return this.fetchEndpoint("starknet_estimateFee", {
       request: [transaction],
+      simulation_flags: [],
       block_id
     }).then(this.responseParser.parseFeeEstimateResponse);
   }
